waku/common/message_test.go

```diff
@@ -187,17 +187,24 @@ func TestMessageSeal(t *testing.T) {
 	target := 32.0
 	params.WorkTime = 4
 	params.PoW = target
-	env.Seal(params) // nolint: errcheck
+	err = env.Seal(params)
+	if err != nil {
+		t.Logf("failed to seal envelope: %s", err)
+	}
```

Comment:
For future debugging it is always good practice to at least log unhandled errors. If there is a breaking change in the future you can see the messages in testing.

Reply:
Agreed.